File: C++/Mahasiswa.cpp
class Mahasiswa :  public SivitasAkademik{
    
    private:
        string nim;
        string fakultas;
        string jurusan;

    public:
    // Constructor
        Mahasiswa(){
            this->nim = "";
            this->fakultas = "";
            this->jurusan = "";
        }

        Mahasiswa(string nim, string fakultas, string jurusan, string asal_univ, string email, string NIK, string name, string gender) : SivitasAkademik(asal_univ, email, NIK, name, gender){
            this->nim = nim;
            this->fakultas = fakultas;
            this->jurusan = jurusan;
        }

    // Setter dan Getter

        // Untuk NIM
        string get_Nim(){
            return this->nim;
        }

        void set_Nim(string nim){
            this->nim = nim;
        }

        // Untuk Fakultas
        string get_Fakultas(){
            return this->fakultas;
        }

        void set_Fakultas(string fakultas){
            this->fakultas = fakultas;
        }

        // Untuk Jurusan
        string get_Jurusan(){
            return this->jurusan;
        }

        void set_Jurusan(string jurusan){
            this->jurusan = jurusan;
        }

        // Destruktor
        ~Mahasiswa(){
        }
    
};

File: Python/Mahasiswa.py
from SivitasAkademik import SivitasAkademik

class Mahasiswa(SivitasAkademik):
    
    # Inisialisasi variabel untuk setiap objek dari kelas
    nim = ""
    fakultas = ""
    jurusan = ""
    
    def __init__(self, nim = "", fakultas = "", jurusan = "", asal_univ = "", email = "", NIK = "", name = "", gender = ""):
        super().__init__(asal_univ, email, NIK, name, gender)
        self.nim = nim
        self.fakultas = fakultas
        self.jurusan = jurusan
        
    # Getter dan Setter untuk Asal Universitas
    def get_Nim(self):
        return self.nim
    
    def set_Nim(self, nim):
        self.nim = nim
        
    # Getter dan Setter untuk Fakultas
    def get_Fakultas(self):
        return self.fakultas
    
    def set_Fakultas(self, fakultas):
        self.fakultas = fakultas
        
    # Getter dan Setter untuk Jurusan
    def get_Jurusan(self):
        return self.jurusan
    
    def set_Jurusan(self, jurusan):
        self.jurusan = jurusan